setup.py

```diff
@@ -151,9 +151,18 @@ def patch_cfitsio(self):
         patches = glob.glob('%s/*.patch' % self.cfitsio_patch_dir)
         for patch in patches:
             fname = os.path.basename(patch.replace('.patch', ''))
-            subprocess.check_call(
-                'patch %s/%s %s' % (self.cfitsio_build_dir, fname, patch),
-                shell=True)
+            try:
+                subprocess.check_call(
+                    'patch -N --dry-run %s/%s %s' % (
+                        self.cfitsio_build_dir, fname, patch),
+                    shell=True)
+            except subprocess.CalledProcessError:
+                pass
+            else:
+                subprocess.check_call(
+                    'patch %s/%s %s' % (
+                        self.cfitsio_build_dir, fname, patch),
+                    shell=True)
```

Comment:
This block fixes the patching.

Reply:
I'll have to take your word for it

Reply:
I tested a bunch of different ways of reinstalling in different orders. It appears to work on my mac at least.